### PR TITLE
Enabled admin for all models

### PR DESCRIPTION
**Pull request by ValentinConstantin**

### CODE REVIEW

File: dv/admin.py [hunk 1]
```diff
@@ -1,281 +1,176 @@
...
+
+        nuts0 = (
+            qs.annotate(code_len=Length('code'))
+            .filter(code_len=2)
+            .values('code', 'label')
+            .distinct()
+        )
+
+        for nn in nuts0:
+            yield nn['code'], '{} ({})'.format(nn['code'], nn['label'])
```
Comment (iuliachiriac) [inline dv/admin.py:55]: Please replace string `format` method with f-strings.

File: dv/admin.py [hunk 2]
```diff
@@ -1,281 +1,176 @@
...
+
+@admin.register(NUTS)
+class NutsAdmin(admin.ModelAdmin):
+    list_display = ('code', 'label')
+    list_filter = (CountryFilter,)
+    search_fields = ('code', 'label')
+    ordering = ('code',)
+
+
+#
```
Comment (iuliachiriac) [inline dv/admin.py:72]: Remove empty comment lines.

File: dv/admin.py [hunk 3]
```diff
@@ -1,281 +1,176 @@
...
+        'state',
+        'programme_area',
+        'financial_mechanism',
+        'gross_allocation',
+    )
+    list_filter = ('state', 'programme_area', 'financial_mechanism')
+    search_fields = (
+        'state__code', 'state__name',
+        'programme_area__code', 'programme_area__name',
+        'financial_mechanism'
```
Comment (iuliachiriac) [inline dv/admin.py:18]: We are storing a financial mechanism code in the database (`EEA`, `NOR`). Please see if there's an easy way to search by the full name of the financial mechanism (`EEA Grants`, `Norway Grants`).

File: dv/admin.py [hunk 4]
```diff
@@ -1,281 +1,176 @@
...
+
+@admin.register(Allocation)
+class AllocationAdmin(admin.ModelAdmin):
+    list_display = (
+        'state',
+        'programme_area',
+        'financial_mechanism',
+        'gross_allocation',
+    )
+    list_filter = ('state', 'programme_area', 'financial_mechanism')
```
Comment (iuliachiriac) [inline dv/admin.py:14]: Please add the funding period as a filter on all admin views (where applicable).

**Review by iuliachiriac — CHANGES_REQUESTED** [review]

File: dv/admin.py (hunk 3 above)
Comment (ValentinConstantin) [reply dv/admin.py:18]: Search by the full name of the financial mechanism already exists.

File: dv/admin.py [hunk 5]
```diff
@@ -1,281 +1,171 @@
...
+    search_fields = (
+        'state__code', 'state__name',
+        'programme_area__code', 'programme_area__name',
+        'financial_mechanism'
+    )
+
+
+@admin.register(Indicator)
+class IndicatorAdmin(admin.ModelAdmin):
+    list_display = ('id', 'indicator')
```
Comment (iuliachiriac) [inline dv/admin.py:24]: The `Indicator` table has changed its structure pretty much. I would add more fields here: 
`list_display = ('funding_period', 'programme', 'programme_area', 'state', 'indicator', 'header')`
The same thing for `OrganisationRole` which has more fields in the current version.

**Review by iuliachiriac — CHANGES_REQUESTED** [review]
The PR looks great! I have another (more general) comment:
- look carefully at all models and include more fields in the `list_display` attribute (usually the "important" fields are placed first in the model definition);
- `list_filter` should include those fields relevant for filtering; the order of the filters is also important. I would include, as filters, the following fields (if they are present), in this order:
    - `funding_period`
    - `financial_mechanism`
    - `priority_sector`
    - `programme_area`
    - `state`
 The other filter fields (specific to each model) should follow these filters.

**Review by iuliachiriac — APPROVED** [review]

**Pull request merged**